docs/developer-guide/nodejs-api.md

```diff
@@ -852,6 +853,40 @@ Example of customizing `RuleTester`:
 ```js
 "use strict";
 
+const RuleTester = require("eslint").RuleTester,
+    test = require("my-test-runner"),
+    myRule = require("../../../lib/rules/my-rule");
+
+let title;
+
+class MyRuleTester extends RuleTester {
+  describe(text, method) {
+    title = text;
+    return method.call(this);
+  }
+
+  it(rule, method) {
```

Comment:
the reason for passing the whole testCase object in, is so that I can give a custom name to the test to support AVA 1s new unique name requirement.